.github/workflows/release_binary.yml

```diff
@@ -0,0 +1,45 @@
+---
+    name: Release Binary
+    
+    on:
+      release:
+        types: [created]
+    
+    permissions:
+      contents: read
+      packages: write
+      
+    jobs:
+      release:
+        permissions: write-all
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout
+            uses: actions/checkout@v3
+          - 
+            name: Set up QEMU
+            uses: docker/setup-qemu-action@v2
+          - 
+            name: Set up Docker Buildx
+            uses: docker/setup-buildx-action@v2
+            
+          - name: Docker compose
```

Comment:
@anhductn2001 why u need to run the application before binary release? kind of validation?

Reply:
I believe it's the simplest approach we can think of at the moment, and it also helps us to verify whether a node can run with the current binary on the Docker environment. Later on, I'll consider expanding it with goreleaser for further development.